### DISCUSSION TITLE
Where are the ffmpeg logs stored

### DISCUSSION THREAD

**Discussion by mgsnell** [blakeblackshear/frigate#2005] [General]
I am new to frigate and am trying to debug an FFMPEG error (crash) when I look at the Frigate NVR log is states the following "[2021-10-13 06:43:45] watchdog.front_porch_cam       ERROR   : FFMPEG process crashed unexpectedly for front_porch_cam.
[2021-10-13 06:43:45] watchdog.front_porch_cam       ERROR   : The following ffmpeg logs include the last 100 lines prior to exit." Were ls "the following ffmpeg logs" located to review?

![logffmpeg error](https://user-images.githubusercontent.com/49707336/137153570-7d0686eb-b26a-4ca4-8d05-f6e00324a1cb.jpg)
?

**Comment by blakeblackshear** [comment]
They are printed right there. All the lines that start with `ffmpeg`. Looks like "corrupt decoded frame..." is why ffmpeg exited.

**Comment by mgsnell** [comment]
Thanks, guess I was being too literal when it said last 100 lines I was looking for more data than that. Now I get to figure out what to do with the "corrupt decoded frame" because as of now what ever the problem was that caused this error corrected itself as it now works
